### ISSUE TITLE
Xavier Initializer's magic number

### ISSUE THREAD

**Issue by pracheer** [zackchase/mxnet-the-straight-dope#249]
Hola,

The tutorials seem to be using Xavier Initializer like this:

> mx.init.Xavier(magnitude=2.24)

But the tutorials don't seem to explain the rational behind 2.24 (at least not until chapter 3) in spite of this being used a lot.

- Pracheer

**Comment by zackchase** [comment]
Good point. Ill discuss it. And maybe rely on it less until it's explained

**Comment by hyuksong** [comment]
should't it be 3 instead of 2.24
